__tests__/getCategories.test.js

```diff
@@ -0,0 +1,40 @@
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const { app } = require("../app");
+
+beforeEach(() => {
+  return seed(testData);
+});
+afterAll(() => {
+  db.end();
+});
+
+describe("app ", () => {
+  describe("/api/categories", () => {
+    test("respond with a status of 200", () => {
+      return request(app).get("/api/categories").expect(200);
+    });
```

Comment:
this first test is a little redundant as you're testing for a status 200 throughout all your tests